data/randomization/randomization-with-weather.ipynb

```diff
@@ -280,9 +280,7 @@
     "print(save_mean['Sum Rain'].min())\n",
```

Comment:
<p>Just an observation: This is pretty interesting. Compare this with the previous same figure (which contains day time hours also), by selecting only the night time files, you can see the proportion of 0 rainfall becomes less.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-echospace/bat-detector-validation/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Yeah, the change in this graph proves that Seattle has more rainfall at night and less rainfall during the day after June!